src/editors/containers/TextEditor/components/EditorFooter/__snapshots__/index.test.jsx.snap

```diff
@@ -2,10 +2,10 @@
 
 exports[`EditorFooter snapshots Save Failed, error message raised 1`] = `
 <div
-  className="editor-footer fixed-bottom"
```

Comment:
That is a merge conflict, plz rectify (fixed-bottom is the bootstrap rule we are using)